### ISSUE TITLE
Ability to paste an image through cntrl V

### ISSUE THREAD

**Issue by ProgrammerIn-wonderland** [cinnyapp/cinny#1243]
### Describe the problem

On other chatting applications (Element, discord, even github issues) you can usually store an image in your clip board and paste it with control V, I use KDE spectacle and have it to automatically copy the image to clip board, it would be nice if pressing cntrl V in cinny had the same behavior as these other apps

### Describe the solution you'd like

Allow cntrl v to paste images, through clipboard API most likely https://web.dev/patterns/clipboard/paste-images/

**Comment by HybridEidolon** [comment]
No, this is an issue with the Tauri configuration in cinny-desktop. Certain permissions need to be enabled in Tauri before they can be used by the web view. Likewise, drag'n'drop is disabled in desktop.

**Comment by iambeingtracked** [comment]
Will it be fixed?
